src/H5AC.c

```diff
@@ -439,34 +439,25 @@ H5AC_dest(H5F_t *f)
 
     /* Check if log messages are being emitted */
     if (H5C_get_logging_status(f->shared->cache, &log_enabled, &curr_logging) < 0)
-
         HGOTO_ERROR(H5E_CACHE, H5E_LOGGING, FAIL, "unable to get logging status")
-    if (log_enabled && curr_logging) {
-
-        if (H5C_log_write_destroy_cache_msg(f->shared->cache) < 0)
-
-            HDONE_ERROR(H5E_CACHE, H5E_LOGGING, FAIL, "unable to emit log message")
-    }
 
     /* Tear down logging */
     if (log_enabled) {
+        if (curr_logging)
+            if (H5C_log_write_destroy_cache_msg(f->shared->cache) < 0)
+                HGOTO_ERROR(H5E_CACHE, H5E_LOGGING, FAIL, "unable to emit log message")
 
         if (H5C_log_tear_down(f->shared->cache) < 0)
-
-            HGOTO_ERROR(H5E_CACHE, H5E_LOGGING, FAIL, "mdc logging tear-down failed")
-    }
+            HGOTO_ERROR(H5E_CACHE, H5E_LOGGING, FAIL, "metadata cache logging tear-down failed")
+    } /* end if */
```

Comment:
I'm not going to die on this hill today, but I consider almost all "end <thing>" comments to be noise. If something is so messy that they are necessary, then that code should be refactored to extract functions, not get poorly-aligned noise comments.

Reply:
Sorry, I like them, but I'm not trying to fight that battle - this slipped in on accident